build/emoji/emoji_gen.mjs

```diff
@@ -342,13 +350,12 @@ export const SkinTranslations = new Map([${skinTranslations.join(', ')}]);
 
 export const ComponentCategory = 'Component';
 
-export const AllEmojiIndicesByCategory = new Map(${JSON.stringify(Array.from(emojiIndicesByCategory))});
```

Comment:
When adding typings for the files generated by this, I wanted to improve encapsulation by not exposing these internal fields. It might be worth precomputing the results of these to avoid holding onto this extra memory going forward